cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -109,13 +110,13 @@ private static String findNetwork(Compute compute, String project) throws IOExce
       NetworkList networkList = compute.networks().list(project).execute();
       networks = networkList.getItems();
     } catch (Exception e) {
-      handleRetryableExceptions(e);
-      throw e;
+      handleRetryableExceptions(e, ErrorTag.CONFIGURATION);
```

Comment:
Over here it's defeinitely not `CONFIGURATION` most of the time. Something like `SocketTimeoutException` is not a configuration

Reply:
Good point. RetryableProvisionException is adding DEPENDENCY and SYSTEM tags. SYSTEM tag needs discussion. Since the errors from handleRetryableException are expected to be transient, the customer can see this as an implementation detail, an issue resulting from communication between the instance and the Dataproc cluster. Hence I went with SYSTEM.